src/mypca.py

```diff
@@ -2,6 +2,7 @@
 
 def main():
     sys.stderr.write("mypca V 1.0.0\n")
+    infilename = sys.argv[1]
```

Comment:
Can we add an output file parameter as well?

Reply:
Sure, no problem!!